### DISCUSSION TITLE
Add Distance to Overview dashboard?

### DISCUSSION THREAD

**Discussion by DrMichael** [teslamate-org/teslamate#3947] [Ideas]
What do you think about adding distance for the selected timeframe to the Overview daskboard?
![image](https://github.com/teslamate-org/teslamate/assets/8087682/87cae58e-dcaf-4ad2-8274-32d1ca000dd5)

**Comment by cwanja** [comment]
Love it! What's the default timeframe again for that dashboard?

**Reply by DrMichael** [reply]
![image](https://github.com/teslamate-org/teslamate/assets/8087682/b1d7f2a6-cd96-4a02-a068-46b660f3fd1e)

**Comment by DrMichael** [comment]
Ok, I gave it a go in PR https://github.com/teslamate-org/teslamate/pull/3949
